chapter_working_incrementally.asciidoc

```diff
@@ -185,6 +185,8 @@ We'll start out very similarly to the first. Edith adds a first item to
 create a to-do list, but we introduce our first new assertion—Edith's
 list should live at its own, unique URL:
 
+// JAN: You could maybe use some other name than Edith. When copy&pasting code I was a bit confused about the comment # Edith starts a new to-do list. I was thinking: "What are we editing here? Oh, look, a typo. I need to report it ... Oh, right - it's a name".
```

Comment:
it's the name of an old family friend so i'm quite attached to it.  but let me at least add a footnote somewhere, reassuring people that it's not a typo.  thanks Jan!